analyzers/src/SonarAnalyzer.CSharp/Rules/SqlKeywordsDelimitedBySpace.cs

```diff
@@ -85,39 +84,52 @@ public sealed class SqlKeywordsDelimitedBySpace : SonarDiagnosticAnalyzer
             .OrderBy(i => i)
             .First();
 
-        protected override void Initialize(SonarAnalysisContext context)
-        {
+        protected override void Initialize(SonarAnalysisContext context) =>
             context.RegisterSyntaxNodeActionInNonGenerated(
                 c =>
                 {
                     var namespaceDeclaration = (NamespaceDeclarationSyntax)c.Node;
-                    var compilationUnit = namespaceDeclaration.Parent as CompilationUnitSyntax;
-                    if (compilationUnit == null ||
-                        (!HasSqlNamespace(compilationUnit.Usings) &&
-                        !HasSqlNamespace(namespaceDeclaration.Usings)))
-                    {
-                        return;
-                    }
-                    var visitor = new StringConcatenationWalker(c);
-                    foreach (var member in namespaceDeclaration.Members)
+                    if (namespaceDeclaration.Parent is CompilationUnitSyntax compilationUnit
+                        && (HasSqlNamespace(compilationUnit.Usings)
+                           || HasSqlNamespace(namespaceDeclaration.Usings)))
                     {
-                        visitor.SafeVisit(member);
+                        var visitor = new StringConcatenationWalker(c);
+                        foreach (var member in namespaceDeclaration.Members)
+                        {
+                            visitor.SafeVisit(member);
+                        }
                     }
-            },
-            SyntaxKind.NamespaceDeclaration);
-        }
+                },
+                SyntaxKind.NamespaceDeclaration);
 
-        private bool HasSqlNamespace(SyntaxList<UsingDirectiveSyntax> usings) =>
+        private static bool HasSqlNamespace(SyntaxList<UsingDirectiveSyntax> usings) =>
             usings.Select(usingDirective => usingDirective.Name)
                 .Any(name => SqlNamespaces.Any(sn => SyntaxFactory.AreEquivalent(name, sn)));
```

Comment:
```suggestion
                usings.Select(x => x.Name)
                    .Any( x => SqlNamespaces.Any(namespace => SyntaxFactory.AreEquivalent(x, namespace ));
```

Reply:
Could you please change also `.Any(name => SqlNamespaces.Any(sn => SyntaxFactory.AreEquivalent(name, sn)));`
to follow the coding style.
It's not super visible from the suggestion but this should also be changed I think to follow the lambda coding style.